Translations/ru.po

```diff
@@ -916,6 +916,7 @@ msgstr "Книга заклинаний"
 msgid "Send Message"
 msgstr "Отправить сообщение"
 
+# Why "умение", but "навык"?
```

Comment:
I think same situation as below, it has to be kept super short to fit the spell square

Reply:
Both words are 5-6 characters, so both will fit in Speedbook
Either word is fine by me. Feel free to pick the word to use throughout